references/cli/def.go

```diff
@@ -772,7 +773,7 @@ func NewDefinitionRenderCommand(c common.Args) *cobra.Command {
 				}
 				config, err := c.GetConfig()
 				if err != nil {
-					return err
+					klog.Infof("ignore kubernetes cluster, unable to get kubeconfig: %s", err.Error())
```

Comment:
which parts of feature in this command rely on this config?

Reply:
So it is due to the usage of legacy PackageDiscover. But it is actually not working now so the config is not required. We will remove it entirely in https://github.com/kubevela/kubevela/pull/6031